pkg/cloud/utils.go

```diff
@@ -18,11 +18,13 @@ package cloud
 
 import (
 	"encoding/json"
+	"errors"
```

Comment:
suggest putting this in its own package.

`pkg/cloud/cerrors/errors.go`

```
package cerrors

func IsGoogleAPINotFound(err error) bool {...}
```

Then the usage will be:

```
cerrors.IsGoogleAPINotFound(err)
```

which should be self documenting.

Reply:
done